pkg/constants/constants.go

```diff
@@ -24,6 +24,12 @@ const (
 	// TODO(#23): Use the kubernetes.io domain when graduating APIs to beta.
 	QueueAnnotation = "kueue.x-k8s.io/queue-name"
 
+	// IgnoreAnnotation is the annotation used to mark workloads to be ignored
```

Comment:
```suggestion
	// IgnoreAnnotation is the annotation used to mark jobs to be ignored
```

In general we might not need to support this annotation for any other CRD.